operator/builtin/input/file/config.go

```diff
@@ -45,20 +46,20 @@ type InputConfig struct {
 	helper.InputConfig `yaml:",inline"`
 	Finder             `mapstructure:",squash" yaml:",inline"`
 
-	PollInterval            helper.Duration        `json:"poll_interval,omitempty"               yaml:"poll_interval,omitempty"`
-	Multiline               helper.MultilineConfig `json:"multiline,omitempty"                   yaml:"multiline,omitempty"`
-	IncludeFileName         bool                   `json:"include_file_name,omitempty"           yaml:"include_file_name,omitempty"`
-	IncludeFilePath         bool                   `json:"include_file_path,omitempty"           yaml:"include_file_path,omitempty"`
-	IncludeFileNameResolved bool                   `json:"include_file_name_resolved,omitempty"  yaml:"include_file_name_resolved,omitempty"`
-	IncludeFilePathResolved bool                   `json:"include_file_path_resolved,omitempty"  yaml:"include_file_path_resolved,omitempty"`
-	StartAt                 string                 `json:"start_at,omitempty"                    yaml:"start_at,omitempty"`
-	FingerprintSize         helper.ByteSize        `json:"fingerprint_size,omitempty"            yaml:"fingerprint_size,omitempty"`
-	MaxLogSize              helper.ByteSize        `json:"max_log_size,omitempty"                yaml:"max_log_size,omitempty"`
-	MaxConcurrentFiles      int                    `json:"max_concurrent_files,omitempty"        yaml:"max_concurrent_files,omitempty"`
-	DeleteAfterRead         bool                   `json:"delete_after_read,omitempty"           yaml:"delete_after_read,omitempty"`
-	LabelRegex              string                 `json:"label_regex,omitempty"                 yaml:"label_regex,omitempty"`
-	Encoding                helper.EncodingConfig  `json:",inline,omitempty"                     yaml:",inline,omitempty"`
-	FilenameRecallPeriod    helper.Duration        `json:"filename_recall_period,omitempty"      yaml:"filename_recall_period,omitempty"`
+	PollInterval            helper.Duration          `mapstructure:"poll_interval,omitempty"               json:"poll_interval,omitempty"               yaml:"poll_interval,omitempty"`
+	Multiline               szhelper.MultilineConfig `mapstructure:"multiline,omitempty"                   json:"multiline,omitempty"                   yaml:"multiline,omitempty"`
```

Comment:
Aspirationally, this should come from the otel repo as well. Is it just out of date in this repo?

Reply:
Agreed, I decided against it for the time being as there is a follow up ticket to get the file input operator merged with otel. To use the otel multiconfig was a large lift and not worth it if we are just going to merge our "upstream" changes into otel and throw this away before `v2.0.0` release.